text/0003-measure-metric-type.md

```diff
@@ -2,47 +2,145 @@
 
 **Status:** `proposed`
 
-## Forward
+# Foreward
 
 This propsal was originally split into three semi-related parts. Based on the feedback, they are now combined here into a single proposal. The original proposals were:
 
     000x-metric-pre-defined-labels
     000x-metric-measure
     000x-eliminate-stats-record
 
-## Overview
+### Updated 8/27/2019
 
-Introduce a `Measure` type of metric object that supports a `Record` API.  Like existing `Gauge` and `Cumulative` metrics, the new `Measure` metric supports pre-defined labels.  A new measurement batch API is introduced for recording multiple metric observations simultaneously.
+A working group convened on 8/21/2019 to discuss and debate the two metrics RFCs (0003 and 0004) and several surrounding concerns.  This document has been revised with related updates that were agreed upon during this working session.  See the (meeting notes)[https://docs.google.com/document/d/1d0afxe3J6bQT-I6UbRXeIYNcTIyBQv4axfjKF4yvAPA/edit#].
```

Comment:
It should be `[meeting notes](https://…)` (parentheses are mixed up).

The access to the document seems to be limited, though (I could not see it without requesting permission).